src/GatewayChecksum.php

```diff
@@ -10,9 +10,9 @@
 
 class GatewayChecksum
 {
-    private const VERSION = "P8.14";
+    private const VERSION = "P8.16";
```

Comment:
Is there a reason we update the version to P8.16 and skip P8.15 ?

Reply:
**P8.15** is used in a merge request currently queued for release.